test/Microsoft.AspNet.Mvc.Razor.Test/Compilation/CompilerCacheTest.cs

```diff
@@ -103,9 +103,8 @@ private static Stream GetMemoryStream(string content)
         }
 
         [Theory]
-        [InlineData(10000)]
+        [InlineData(10000)] // expected failure: same time and length (PranavK)
```

Comment:
It comes from here - https://github.com/aspnet/Mvc/commit/6600e68fc0d64e382f9172bfc4da1cbcb12bee10#diff-64a550e80195e44d72c7cba6ea99f428R107. I think you can drop it - the test name should suffice in explaining what scenario this is.